src/server/database/migrations/20170522164500-markdownToHTML.js

```diff
@@ -1,87 +1,92 @@
-import {convertFromHTML, ContentState, convertToRaw, convertFromRaw, SelectionState} from 'draft-js';
-import MarkdownIt from 'markdown-it';
-import emoji from 'markdown-it-emoji';
-import jsdom from 'jsdom';
-import toMarkdown from 'to-markdown';
-import {stateToHTML} from 'draft-js-export-html';
-import entitizeText from 'universal/utils/draftjs/entitizeText';
-import getTagsFromEntityMap from 'universal/utils/draftjs/getTagsFromEntityMap';
+/*
+* These packages relied on things that were part of react 15 but not react 16, so they are all broken
```

Comment:
Sweet comment! 😎  I'm wondering though why react should be affecting our DB migrations?

Reply:
because the world doesn't make sense... i think it was `draft-js-export-html` that relied on something that lived in react-dom. for the backstory: our project cards initially were just text areas, then they were text areas that accepted markdown & when you clicked away from them the markdown compiled, then we moved to draft-js. to convert markdown to draft-js you need to go markdown -> html -> draftjs.

Reply:
So what format is the text saved in these days? Markdown?

Reply:
a utf-8 stringified json, just for kicks dive into the DB & dig up the `content` field. Then edit a project to have an @ mention, a tag, an emoji, and see how the shape changes.